nltk/tokenize/destructive.py

```diff
@@ -62,6 +62,7 @@ class NLTKWordTokenizer(TokenizerI):
         (re.compile("([»”’])", re.U), r" \1 "),
         (re.compile(r"''"), " '' "),
         (re.compile(r'"'), " '' "),
+        (re.compile(r"\s+"), " "),
```

Comment:
This will compress the continuous whitespaces of different types into one. 

Note: `\s` = `[\r\n\t\f\v ]`